spyvm/model.py

```diff
@@ -413,8 +413,6 @@ def clone(self, space):
         return W_LargePositiveInteger1Word(self.value)
 
     def at0(self, space, index0):
-        if index0 >= self.size():
```

Comment:
Because this is only called from here: https://github.com/HPI-SWA-Lab/RSqueak/blob/tim/block-activation/spyvm/primitives.py#L484. There is already an index check